easygcm/src/main/java/eu/inloop/easygcm/GcmHelper.java

```diff
@@ -185,30 +186,15 @@ void registerInBackground(final Context context, final RegistrationListener regi
         final AsyncTask<Void, Void, Void> registrationTask = new AsyncTask<Void, Void, Void>() {
```

Comment:
Good point, the current synchronization is enough. How about replacing AsyncTask with an IntentService as in the sample? I think it's nicer and less likely to be killed.